strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py

```diff
@@ -107,7 +108,7 @@ def on_request_accepted(self) -> None:
 
     async def handle_connection_init_timeout(self) -> None:
         task = asyncio.current_task()
-        assert task
+        assert task is not None  # for typecheckers
```

Comment:
nit: since we enforce typing a lot here, `assert <something>` is a well known practice for type checkers, so I don't think the comment is required

```suggestion
        assert task is not None
```